audio_to_spectrogram/scripts/spectrum_plot.py

```diff
@@ -35,10 +57,16 @@ def _cb(self, msg):
         self.ax.set_xlim((self.freq.min(), self.freq.max()))
         self.ax.set_ylim((0.0, 20))
         self.ax.legend(loc='upper right')
+        print(self.pub_img.get_num_connections())
+        if self.pub_img.get_num_connections() > 0:
```

Comment:
How about using `ConnectionBasedTransport`?
https://jsk-docs.readthedocs.io/projects/jsk_common/en/latest/jsk_topic_tools/class/python_connection_based_transport.html

Reply:
That's good idea, thanks. I modified the code. https://github.com/jsk-ros-pkg/jsk_recognition/pull/2654/commits/2e86235c63a0d80d6497db22606e8940ef1bbaf2